### DISCUSSION TITLE
Scan video ID's from multiple channels

### DISCUSSION THREAD

**Discussion by QuentinMcCarthy** [ThioJoe/YT-Spammer-Purge#612] [Ideas]
When listing video ID's to scan, why is it that the program can only scan listed ID's from a single channel? Why not make it so that it can scan ANY channel's video ID's because otherwise you're having to restart scans for every channel you scan.

I'm not sure what the limitation is.

I'm asking about it because I scan videos as I go through watching them in my Watch Later list, and it'd be nice to be able to scan all the videos I've watched at once instead of one at a time (they're not all from the same channel, for obvious reasons)

**Comment by ThioJoe** [comment]
The limitation is because the program has to store info about the channel owner being scanned to prevent their comments from being flagged by mistake. Theoretically I could make it possible to scan multiple channels but it would require rewriting potentially a decent amount of stuff

**Reply by QuentinMcCarthy** [reply]
Ah I see. It just seems like a rather odd limitation.
